src/pages/pattern-library.tsx

```diff
@@ -145,6 +146,20 @@ function Page() {
               </li>
             </ul>
           </section>
+
+          <section className="grid-section grid-content">
+            <h2 className="title-subsection">Custom Button – Primary</h2>
+            <ul className="list-horizontal">
+              <li>
+                <Button variant="primary">Add to Cart</Button>
+              </li>
+              <li>
+                <Button variant="primary" disabled>
+                  Add to Cart
+                </Button>
+              </li>
+            </ul>
+          </section>
```

Comment:
I wonder if we should place this items in the first button's `grid-section`.

Reply:
@renatamottam will update these whole section later 🤩